packages/visual-dev/CHANGELOG.md

```diff
@@ -25,14 +25,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Components changed
   - \<RadioCard>
-    - Refactored `RadioCardGroupView` to now use dot notation `RadioCardView.GroupView`
-    - Updated title & description props to now allow for `string | React.ReactNode`
-    - Removed unnecessary ternary conditionals
-    - Changed `min-height` from `138px` to `95px`
+    <<<<<<< HEAD - Added more flexibility to allow custom CSS and better design for cards without icons - Added a disabled state
```

Comment:
We also need to make sure that we remember to add git tags at the bottom of the CHANGELOG